CONTRIBUTING.md

```diff
@@ -31,21 +31,23 @@ What follows are recommendations/requirements to keep in mind while contributing
 ## Making changes
 
 ### Adding tests
+
 When modifying and/or adding new code it is important to ensure the changes are covered by tests.
 Test thoroughly, but not excessively.
 Mitiq uses a nested structure for packaging tests in directories named `tests` at the same level of each module.
-The only except to this is that any tests requiring a QVM should be placed in the `mitiq_pyquil/tests` folder.
+The only exception to this is that any tests requiring a QVM should be placed in the `mitiq_pyquil/tests` folder.
 
 ### Running tests
 
-After making changes, please ensure your changes pass all the existing tests (and any new tests you've added).
-You can run the tests using the `pytest` CLI, but the [Makefile][makefile] contains many of the common commands you'll need to ensure your code is aligned with our standards.
+After making changes, ensure your changes pass all the existing tests (and any new tests you've added).
+Use `pytest mitiq/$MODULE` to run the tests for the module you are working on.
+Once they pass, you can run the entire test suite (excluding those that require the pyQuil QVM) by running the following command.
 
-For example, to run all the tests that do not require a pyQuil QVM, run
 ```bash
 make test
 ```
-This is typically suitable for most development tasks and is the easiest, and most common way to test.
+
+This can often be slow, however, so testing your changes iteravely using `pytest` is often faster when doing development.
```

Comment:
```suggestion
This can often be slow, however, so testing your changes iteratively using `pytest` is often faster when doing development. Inside the Mitiq repository, navigate to the `tests` folder of interest (`cd mitiq/mitiq/some_module/tests`).  

If there are multiple test files in this folder & you want to run all the tests, `(path to some tests folder) pytest` will do the job. But if you want to run the tests in only one specific file `(path to some tests folder) pytest test_some_feature.py` will work. 
```

Reply:
I think this is explained on lines 42/43. Do you think we need this as well?